test/unit/adapters/proposals-quert.spec.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2017 The "mysteriumnetwork/mysterium-vpn" Authors.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// @flow
+
+import ProposalsQuery from '../../../src/adapters/proposals-query'
+
+describe('ProposalsQuery', () => {
+  describe('toQueryParams', () => {
+    it('defaults to fetching metrics if nothing was passed to constructor', () => {
+      const q = new ProposalsQuery()
+      expect(q.toQueryParams()).to.eql({fetchConnectCounts: true})
```

Comment:
Why to have 2nd layers of defaults (Tequilapi API & js-tequilapi client)?
Dont overcomplicate this lib with features we dont need - make same defaults as in Tequilapi API.